cmd/gctcli/commands.go

```diff
@@ -2805,3 +2806,112 @@ func clearScreen() error {
 		return cmd.Run()
 	}
 }
+
+const timeFormat = "2006-01-02 15:04:05"
+
+var startTime, endTime, order string
+var limit int
+
+var getAuditEventCommand = cli.Command{
+	Name:   "getauditevent",
+	Usage:  "gets audit events matching query parameters",
```

Comment:
Please add ArgsUsage here, helpful to know the order of commands if they don't want to use the full flag names